### ISSUE TITLE
Changes to Modal Presentation Style

### ISSUE THREAD

**Issue by Errortype520** [kevin0571/STPopup#78]
The changes made to modal presentation style, from UIModalPresentationCustom to UIModalPresentationOverCurrentContext seems to have broken most, but not all of my popups.

It works from any view controller I have presented modally, but not on any root view controllers. I am at a loss for why.

**Comment by kevin0571** [comment]
@Errortype520 broken means the popup is not showed at all?

**Comment by Errortype520** [comment]
Yes, I am unsure why, but switching this back to UIModalPresentationCustom keeps my popups showing.

**Comment by Errortype520** [comment]
It is possible it is a conflict with SlideMenuController or another library I am using

**Comment by jakubheglas** [comment]
I second this one. It'd be great to make this either a public property or make it .overFullScreen (which imo makes more sense for far more use cases). Thoughts?
